### DISCUSSION TITLE
Generate x509 v2 certificate

### DISCUSSION THREAD

**Discussion by carlosadrianosousa** [phpseclib/phpseclib#1844] [Q&A]
Hi everyone.

Is it possible to generate a v2 x509 certificate using phpseclib 3?

**Comment by terrafrost** [comment]
Currently, only v3 X509 certs are supported.

What's the use case for v2 X509 certs? idk what the differences between v2 and v3 are so idk how involved it'd be to add v2 support but knowing the use case might help motivate me!
